region/brinstar/red.json

```diff
@@ -1199,6 +1224,26 @@
                     "Which frame is determined by actual shinesparkFrames and shinespark velocity."
                   ]
                 },
+                {
+                  "name": "Shinespark (Come In Shinecharging)",
+                  "notable": false,
+                  "entranceCondition": {
+                    "comeInShinecharging": {
+                      "length": 6,
+                      "openEnd": 1
+                    }
+                  },
+                  "requires": [
+                    {"shinespark": {
+                      "frames": 150,
```

Comment:
146 frames through the door, 60 excess frames
at the end of the runway is 142 frames, with 60 excess